hazelcast/src/main/java/com/hazelcast/internal/metrics/MetricDescriptorConstants.java

```diff
@@ -199,6 +207,7 @@
     public static final String MAP_PREFIX = "map";
     public static final String MAP_PREFIX_INDEX = "map.index";
     public static final String MAP_PREFIX_NEARCACHE = "map.nearcache";
+    public static final String MAP_PREFIX_ENTRY_PROCESSOR_OFFLOADABLE_EXECUTOR = "map.entry.processor.offloadable.executor";
```

Comment:
Should this be under executors instead? I see other internal executors like `executor.durable.internal`.

Reply:
Statistics collection for that is enabled/disabled per map, this was the reason i did it like this. WDYT?

Reply:
Ok, let's keep it here but maybe just change it to `map.entryprocessor.offloadable.executor` or change it even to `map.offloadable.executor` because:
• we have other prefixes that don't separate words with dots (e.g. nearcache)
• maybe we will use that executor for other tasks so we can remove mention of EP